src/main/resources/compile_invalids_wrapper_fail_on_error.sql

```diff
@@ -104,12 +104,15 @@ begin
 end;
 /
 
+set heading off
 select 'Invalid objects in [' || user || ']:'
 from dual;
+
 column object_type format a15
 column object_name format a30 wra
 column invalid_message format a15
 set tab off
+set heading off
```

Comment:
should be `set heading on` ?

Reply:
Then the header will be duplicated for each object.
![image](https://user-images.githubusercontent.com/43806989/109774190-e7d20d80-7c21-11eb-895f-910e2be5944b.png)